app/src/test/java/io/apicurio/registry/ibmcompat/IBMCompatApiTest.java

```diff
@@ -376,8 +405,8 @@ public void testPatchSchemaVersionState() throws Exception {
                 .patch("/ibmcompat/schemas/" + schemaName + "/versions/2")
             .then()
                 .statusCode(200)
-                .body("name", equalTo(schemaName))
-                .body("id", equalTo(schemaName))
+                .body("name", equalTo(schemaId))
+                .body("id", equalTo(schemaId))
                 .body("enabled", is(true))
                 .body("state.state", equalTo("active"))
                 .body("versions.size()", is(2))
```

Comment:
Adding the new version might be an async operation - do we need a check *before* this Patch operation to make sure the version added above exists and is in the expected state?

Reply:
Good idea - I've added a waitForVersion method & used it where appropriate.